lib/cli.js

```diff
@@ -877,6 +877,7 @@ exports.search = function(keywords) {
 
 exports.init = function(options) {
   if (!shell.which('git')) return console.log(chalk.red('    Prerequisite not installed: git'));
+  if (!shell.which('mongod')) return console.log(chalk.red('    Prerequisite not installed: mongod'));
```

Comment:
@BrockMcKean thanks for working on this, but I must I agree. Let's just hold off for now.